src/timeseriesflattener/feature_specs/single_specs.py

```diff
@@ -0,0 +1,198 @@
+from dataclasses import dataclass
+from typing import Callable, Optional, Union
+
+import pandas as pd
+from pydantic import Field
+from timeseriesflattener.aggregation_functions import concatenate
+from timeseriesflattener.utils.pydantic_basemodel import BaseModel
+
+
+@dataclass(frozen=True)
+class CoercedFloats:
+    lookwindow: Union[float, int]
+    fallback: Union[float, int]
+
+
+def coerce_floats(lookwindow: float, fallback: float) -> CoercedFloats:
+    lookwindow = lookwindow if not lookwindow.is_integer() else int(lookwindow)
+    fallback = fallback if not fallback.is_integer() else int(fallback)
+
+    return CoercedFloats(lookwindow=lookwindow, fallback=fallback)
+
+
+class StaticSpec(BaseModel):
+    """Specification for a static feature.
+
+    Args:
+        base_values_df: Dataframe with the values. Should contain columns named "entity_id", "value" and "timestamp".
```

Comment:
```suggestion
        base_values_df: Dataframe with the values. Should contain columns containing the "entity_id", "value" and "timestamp". The names of these columns is specified when calling TimeSeriesFlattener.XX
```
We should update this so there's no confusion on what types of columns are required, and where  you specify the name of the columns